src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -40,4 +44,30 @@ TranscriptionUrgencyResponse mapToTranscriptionUrgencyResponse(TranscriptionUrge
         transcriptionUrgencyResponse.setDescription(transcriptionUrgencyEntity.getDescription());
         return transcriptionUrgencyResponse;
     }
+
+    public static TranscriptionResponse mapToTranscriptionResponse(TranscriptionEntity transcriptionEntity) {
+
+        TranscriptionResponse transcriptionResponse = new TranscriptionResponse();
+        try {
+            transcriptionResponse.setCaseId(transcriptionEntity.getCourtCase().getId());
+            transcriptionResponse.setCaseNumber(transcriptionEntity.getCourtCase().getCaseNumber());
```

Comment:
If we need to navigate to another entity relationship to get some details, we should probably introduce a variable rather than call e.g. getCourtCase() each time?

Reply:
Refactored code to get court case variable